test/test_game.rb

```diff
@@ -7,19 +7,35 @@
 
   it 'checks a right guess' do
     word.expects(:guess).with("A").returns(true)
-    word.expects(:uncovered_word).returns(uncovered_word)
-    uncovered_word.expects(:include?).returns(true)
+    word.expects(:completed?).returns(false)
     game = Game.new(word)
     game.guess("A")
     game.try.must_equal 11
   end
 
   it 'checks a wrong guess' do
     word.expects(:guess).with("A").returns(false)
-    word.expects(:uncovered_word).returns(uncovered_word)
-    uncovered_word.expects(:include?).returns(true)
+    word.expects(:completed?).returns(false)
     game = Game.new(word)
     game.guess("A")
     game.try.must_equal 10
   end
+
+  it 'checks for loose' do
```

Comment:
it's "lose", not "loose". but I'd call it "lost?" anyway (and replace "win?" with "won?").